### DISCUSSION TITLE
related resource relation field

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#82] [Q&A] [answered]
I have been noticing a variety in the way the "Relation" field is used in "Related resources" objects. This is causing problems for me because I am using these relations to find dandisets that are associated with publications to report on milestones for the NWB U24. My initial approach was to look for the relation "IsDescribedBy" and a DOI, but looking through the data, I am finding that either I am being too restrictive or several datasets are using the "Relation" field incorrectly because I am missing dandisets that clearly should meet the criterion of having an associated publication.

* https://dandiarchive.org/dandiset/000147 is an example of a dataset that uses "dcite:IsDescribedBy", as I would expect.
* https://dandiarchive.org/dandiset/000212 lists 3 papers that all use the "dcite:Describes" relation. I would suspect these papers describe the data, so I would have used "dcite:IsDescribedBy"
* https://dandiarchive.org/dandiset/000231 also uses "Describes"
* https://dandiarchive.org/dandiset/000026 (while not NWB) uses "dcite:IsPublishedIn"

What is the right relation to use here? I think it's unclear whether the dandiset is supposed to be the subject or the object of the sentence, which is causing some confusion. Maybe we could add an explanatory note to the metadata page.

**Comment by satra** [comment, marked as answer]
i think this exists, but people are not paying attention.

<img width="542" alt="image" src="https://user-images.githubusercontent.com/184063/199602576-fafcf057-edcc-4496-98b9-b841dcd2a539.png">

**Reply by bendichter** [reply]
ok yes this is good. So to be clear, for the primary paper in which a dataset is described, is the preferred relation "IsDescribedBy" (as opposed to "IsPublishedIn", or some other relation type?)

**Reply by satra** [reply]
correct.
